src/librustc_mir/borrow_check/nll/explain_borrow/mod.rs

```diff
@@ -286,66 +286,62 @@ impl<'cx, 'gcx, 'tcx> MirBorrowckCtxt<'cx, 'gcx, 'tcx> {
     }
 
     /// Check if a borrow location is within a loop.
```

Comment:
Can we update the comment? Something like "true if the `borrow_location` can reach the `use_location` without passing through any backedges."